ppxlib-bench.opam

```
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Run ppxlib benchmarks"
+description: """
+Third-party code in benchmarks depends on later versions of the ocaml
+than ppxlib itself. Also the benchmark runner has dependencies that ppxlib doesn't
+have."""
+maintainer: ["opensource@janestreet.com"]
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+license: "MIT"
+homepage: "https://github.com/ocaml-ppx/ppxlib"
+doc: "https://ocaml-ppx.github.io/ppxlib/"
+bug-reports: "https://github.com/ocaml-ppx/ppxlib/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.04.1"}
+  "ppxlib" {= version}
+  "base"
```

Could you add this for consistency with the `dune-project` file?

```suggestion
  "base" {>= "v0.15" & < "v0.16"}
```

---

I ended up removing the versions from `dune-project` instead because they were causing CI to limit the versions of ocaml used for testing to just those compatible with that range of versions.